build/ci/postInstall.js

```diff
@@ -103,6 +103,29 @@ function makeVariableExplorerAlwaysSorted() {
     }
 }
 
+function fixJupyterLabRenderers() {
+    const warnings = [];
+    ['node_modules/@jupyterlab/cells/lib/widget.js', 'node_modules/@jupyterlab/rendermime/lib/renderers.js'].forEach(
+        (file) => {
+            const filePath = path.join(__dirname, '..', '..', file);
+            if (!fs.existsSync(filePath)) {
+                return;
+            }
+            const textToReplace = `import marked from 'marked'`;
+            const textToReplaceWith = `import { marked } from 'marked'`;
```

Comment:
The way `marked` had to be imported changed in v3.0 (we were using <1 and now we're using v4)
Jupyter lab too has updated their packages, however they haven't published a stable version yet, hence we have to do this.